latest/template_all_features.yaml

```diff
@@ -7,10 +7,10 @@ Parameters:
   SplunkAccessToken:
     Type: String
     NoEcho: true
-    Description: "Copy your Splunk Observability access token from the Observability Cloud AWS integration wizard. You can also find it in Profile > Organization Settings > Access Tokens."
+    Description: "Copy your Splunk Observability access token having INGEST authorization scope from Settings > Access Tokens."
```

Comment:
"having INGEST" --> "with INGEST"

Reply:
no, "with" is "copy with", having is having